pkgs/applications/audio/clementine/default.nix

```diff
@@ -22,9 +63,24 @@ let
 
   patches = [
     ./clementine-spotify-blob.patch
+    (fetchpatch { # fix execution on wayland (1.4.0rc1-131-g2179027a6)
```

Comment:
Please add a reference to the upstream issue as well. It's nice to have that context if someone wants to look into the background of the patch.

Reply:
right!

Reply:
done